nuxt.config.ts

```diff
@@ -94,4 +136,6 @@ export default defineNuxtConfig({
     shim: false,
     strict: true,
   },
+
+  vite: viteConfig,
```

Comment:
Is this config option new? I assume this is why you pulled the config from it's own file?

Reply:
Ya, they deprecated having `vite.config` in the nuxt project, i assume there's some extra type or other info they can get with it being in the nuxt config itself for build or other steps.